angr/storage/kvstore.py

```diff
@@ -0,0 +1,41 @@
+
+from .memory import SimMemory
+
+
+class TypedVariable(object):
```

Comment:
`(object)` is not necessary in Python 3.

Reply:
Fixed Now